orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/job/RunJobTask.groovy

```diff
@@ -43,12 +46,15 @@ class RunJobTask extends AbstractCloudProviderAwareTask implements RetryableTask
   long timeout = 60000
 
   @Override
-  void onTimeout(Stage stage) {
+  @Nullable
+  TaskResult onTimeout(@Nonnull Stage stage) {
     jobUtils.cancelWait(stage)
+
+    return null;
   }
 
   @Override
-  void onCancel(Stage stage) {
+  void onCancel(@Nonnull Stage stage) {
```

Comment:
is adding `@Nonnull` actually buying us anything? I was operating under the assumption that if something isn't `@Nullable` then it has to be `@Nonnull`

Reply:
but this is groovy, isn't everything `nullable`? Mostly, I added this because intellij complains when the base method has the annotation and the methods in derived classes don't